### PR TITLE
Feature 2 Added along with two files show and hide

### PR DESCRIPTION
**Pull request by rajeshrajkr4**
I have added a new feature which will show and hide folder

### CODE REVIEW

**Review by rahul37865 — CHANGES_REQUESTED** [review]

File: feature2/hide.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
[the remark this answers is not in the document]
Comment (rajeshrajkr4) [reply feature2/hide.html]: I have done change in hide file title

File: feature2/show.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
[the remark this answers is not in the document]
Comment (rajeshrajkr4) [reply feature2/show.html]: I have done change in show file title

File: feature2/show.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+</head>
+
+<body>
+  <h1>This will show folder</h1>
```
[the remark this answers is not in the document]
Comment (rajeshrajkr4) [reply feature2/show.html:12]: Line added using lorem

**Review by rahul37865 — APPROVED** [review]
OK all done good job raj

**Pull request merged**